### ISSUE TITLE
Crash on launch using Curse Launcher

### ISSUE THREAD

**Issue by aliatic** [MJRLegends/Space-Astronomy-Feedback-#15]
I've downloaded the pack on the Curse launcher, but when I try and play I get this error log (below). 

Not sure how to fix it. 
I've played other packs on Curse fine, but I have been having a couple of issues with Curse and Java recently (I'm still playing packs on FTB fine). 
I've already updated my Java to the latest version. 
Not sure what else to do?

[20:39:27] [main/INFO] [LaunchWrapper]: Loading tweak class name cpw.mods.fml.common.launcher.FMLTweaker
[20:39:27] [main/INFO] [LaunchWrapper]: Using primary tweak class name cpw.mods.fml.common.launcher.FMLTweaker
[20:39:27] [main/INFO] [LaunchWrapper]: Calling tweak class cpw.mods.fml.common.launcher.FMLTweaker
[20:39:28] [main/INFO] [FML]: Forge Mod Loader version 7.99.30.1492 for Minecraft 1.7.10 loading
[20:39:28] [main/INFO] [FML]: Java is Java HotSpot(TM) 64-Bit Server VM, version 1.8.0_25, running on Windows 10:amd64:10.0, installed at D:\Documents\Curse\Minecraft\Install\runtime\jre-x64\1.8.0_25
[20:39:28] [main/INFO] [FML]: [AppEng] Core Init
[20:39:28] [main/WARN] [FML]: The coremod aroma1997.core.coremod.CoreMod does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[20:39:28] [main/WARN] [FML]: The coremod codechicken.core.launch.CodeChickenCorePlugin does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[20:39:28] [main/INFO] [STDERR]: [codechicken.core.launch.DepLoader$DepLoadInst:checkExisting:436]: Warning: version of bspkrsCore, 6.16 is newer than request 6.15
[20:39:28] [main/WARN] [FML]: The coremod cofh.asm.LoadingPlugin does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[20:39:28] [main/WARN] [FML]: The coremod fastcraft.LoadingPlugin does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[20:39:28] [main/WARN] [FML]: The coremod codechicken.core.launch.DepLoader does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[20:39:28] [main/WARN] [FML]: The coremod mrtjp.relocation.asm.RelocationPlugin does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[20:39:28] [main/WARN] [FML]: The coremod mods.immibis.core.ICCoreMod does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[20:39:28] [main/WARN] [FML]: The coremod ic2.core.coremod.IC2core does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[20:39:28] [main/WARN] [FML]: The coremod invtweaks.forge.asm.FMLPlugin does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[20:39:28] [main/WARN] [FML]: The coremod logisticspipes.asm.LogisticsPipesCoreLoader does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[20:39:28] [main/WARN] [FML]: The coremod mekanism.common.asm.LoadingHook does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[20:39:28] [main/WARN] [FML]: The coremod micdoodle8.mods.miccore.MicdoodlePlugin does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: java.lang.NoClassDefFoundError: scala/collection/Seq
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.lang.Class.forName0(Native Method)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.lang.Class.forName(Class.java:344)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.relauncher.CoreModManager.loadCoreMod(CoreModManager.java:460)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.relauncher.CoreModManager.discoverCoreMods(CoreModManager.java:389)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.relauncher.CoreModManager.handleLaunch(CoreModManager.java:221)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.relauncher.FMLLaunchHandler.setupHome(FMLLaunchHandler.java:90)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.relauncher.FMLLaunchHandler.setupClient(FMLLaunchHandler.java:67)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.relauncher.FMLLaunchHandler.configureForClientLaunch(FMLLaunchHandler.java:34)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.common.launcher.FMLTweaker.injectIntoClassLoader(FMLTweaker.java:126)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:115)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: Caused by: java.lang.ClassNotFoundException: scala.collection.Seq
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.net.URLClassLoader$1.run(URLClassLoader.java:372)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.security.AccessController.doPrivileged(Native Method)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:117)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	... 11 more
[20:39:28] [main/ERROR] [FML]: An error occurred trying to configure the minecraft home at D:\Documents\Curse\Minecraft\Instances\Space Astronomy for Forge Mod Loader
java.lang.NoClassDefFoundError: scala/collection/Seq
	at java.lang.Class.forName0(Native Method) ~[?:1.8.0_25]
	at java.lang.Class.forName(Class.java:344) ~[?:1.8.0_25]
	at cpw.mods.fml.relauncher.CoreModManager.loadCoreMod(CoreModManager.java:460) ~[forge-1.7.10-10.13.4.1492-1.7.10.jar:?]
	at cpw.mods.fml.relauncher.CoreModManager.discoverCoreMods(CoreModManager.java:389) ~[forge-1.7.10-10.13.4.1492-1.7.10.jar:?]
	at cpw.mods.fml.relauncher.CoreModManager.handleLaunch(CoreModManager.java:221) ~[forge-1.7.10-10.13.4.1492-1.7.10.jar:?]
	at cpw.mods.fml.relauncher.FMLLaunchHandler.setupHome(FMLLaunchHandler.java:90) [forge-1.7.10-10.13.4.1492-1.7.10.jar:?]
	at cpw.mods.fml.relauncher.FMLLaunchHandler.setupClient(FMLLaunchHandler.java:67) [forge-1.7.10-10.13.4.1492-1.7.10.jar:?]
	at cpw.mods.fml.relauncher.FMLLaunchHandler.configureForClientLaunch(FMLLaunchHandler.java:34) [forge-1.7.10-10.13.4.1492-1.7.10.jar:?]
	at cpw.mods.fml.common.launcher.FMLTweaker.injectIntoClassLoader(FMLTweaker.java:126) [forge-1.7.10-10.13.4.1492-1.7.10.jar:?]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:115) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
Caused by: java.lang.ClassNotFoundException: scala.collection.Seq
	at java.net.URLClassLoader$1.run(URLClassLoader.java:372) ~[?:1.8.0_25]
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361) ~[?:1.8.0_25]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_25]
	at java.net.URLClassLoader.findClass(URLClassLoader.java:360) ~[?:1.8.0_25]
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:117) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_25]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_25]
	... 11 more
Exception in thread "main" [20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: java.lang.NoClassDefFoundError: scala/collection/Seq
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.lang.Class.forName0(Native Method)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.lang.Class.forName(Class.java:344)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.relauncher.CoreModManager.loadCoreMod(CoreModManager.java:460)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.relauncher.CoreModManager.discoverCoreMods(CoreModManager.java:389)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.relauncher.CoreModManager.handleLaunch(CoreModManager.java:221)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.relauncher.FMLLaunchHandler.setupHome(FMLLaunchHandler.java:90)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.relauncher.FMLLaunchHandler.setupClient(FMLLaunchHandler.java:67)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.relauncher.FMLLaunchHandler.configureForClientLaunch(FMLLaunchHandler.java:34)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.common.launcher.FMLTweaker.injectIntoClassLoader(FMLTweaker.java:126)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:115)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: Caused by: java.lang.ClassNotFoundException: scala.collection.Seq
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.net.URLClassLoader$1.run(URLClassLoader.java:372)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.security.AccessController.doPrivileged(Native Method)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:117)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[20:39:28] [main/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	... 11 more

**Comment by aliatic** [comment]
I've tried installing it on the FTB launcher, and it starts to load up after installation, but then I get an error message saying: "java platform se binary has stopped working". 

I'm understanding that this is a Java issue - but I don't know how to fix it. I've made sure I have the latest Java (v8) installed, both 32 and 64 bit. I also tried installing v7, but that doesn't seem to clear anything up. 

Other packs on the same launcher ARE working, so I don't understand why this pack isn't working with my machine?
